### ISSUE TITLE
Cors error on contribute submission

### ISSUE THREAD

**Issue by sabinevidal** [unicef/kindly-website#13]
When submitting a response on the [contribute](https://kindly.unicef.io/contribute) page, the following error is shown in the console. Likely related to Azure as mentioned, as in #11 
 
```
Access to fetch at 'https://script.google.com/macros/s/...' from origin 'https://kindly.unicef.io' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

**Comment by lacabra** [comment]
Ah, this has nothing to do with Azure. This is related to Google App Scripts. It is likely that @nathanfletcher has redeployed an App Script changing permissions. We encountered this error in the past.

**Comment by lacabra** [comment]
@sabinevidal, @nathanfletcher can you provide an update on this? If no longer relevant, I advise to close it.

**Comment by sabinevidal** [comment]
@lacabra Seems to have resolved since I was granted more permissions on GA Scripts.
